sdcm/sct_events.py

```diff
@@ -386,6 +386,53 @@ def eval_filter(self, event):
         return result
 
 
+class GroupCommonDbEvents:
+    @contextmanager
```

Comment:
can you check if those can be at least classmethods, so you don't need to instantiate  this object each time ?

Reply:
I can create a singleton object so that there is only a single instance throughout the run.